server/middleware/gqlAuth/index.js

```diff
@@ -43,37 +39,6 @@ export const isPublicQuery = async req => {
   return queries.every(({ queryName, operationType }) => GQL_QUERY_TYPES[operationType].whitelist.includes(queryName));
 };
 
-export const isAuthenticated = async (req, res, next) => {
-  try {
-    // For accessing graphql without authentication when debugging.
-    if (req.method === 'get') {
-      next();
-      return;
-    }
-    if (isTestEnv() || (await isPublicQuery(req))) {
-      next();
-    } else {
-      let args;
-      const graphQLBody = gql`
-        ${req.body.query}
-      `;
-      if (graphQLBody.definitions?.length && graphQLBody.definitions[0].selectionSet?.selections?.length) {
-        args = convertToMap(graphQLBody.definitions[0].selectionSet.selections[0].arguments, req.body.variables);
-        req.parentArgs = args;
```

Comment:
if we remove this won't everything downstream break, cause we don't have pparentAgs?